samples/mcuboot/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _mcuboot:
```

Comment:
There's already a sample with this tag name (they all need to be unique within the documentation set).

Reply:
Done